nemo/core/classes/modelPT.py

```diff
@@ -312,9 +312,16 @@ def val_dataloader(self):
         if self._validation_dl is not None:
             return self._validation_dl
 
-    # def test_dataloader(self):
-    #     if self._test_dl is not None:
-    #         return self._test_dl
+    def validation_step(self, batch, what_is_this_input):
```

Comment:
should we also add such a "patch" to training_step?

use-case: when one will try to write his own pytorch training loop?

Reply:
I think it's not necessary. At the bare minimum all of our Nemo models will have train step defined, and users can define their own train loop inside ptl too. But it's fine to have either way. I'll leave it to the reviewers.

Reply:
wait... train loop inside PTL? what do you mean? would like to learn more, let's take that offline.

Reply:
@okuchaiev Any preference? I don't mind having a default traon_step() or not, so I'll defer to your opinion here.

Reply:
I would add this patch to training_step as well to make it easier to use without lightning's trainer, but I'd leave it to @tkornuta-nvidia  to make a call here

Reply:
ok, added